recipes/pycoverm/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pycoverm" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e5e96e55ee2756d0cbd1db383e136cd2c1e232204f25b015ce956b7a6f4ddee2
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('pycoverm', max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - {{ compiler("rust") }}
+    - {{ stdlib("c") }}
+    - python                              # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - crossenv                            # [build_platform != target_platform]
+    - maturin >=1.7,<2                    # [build_platform != target_platform]
```

Comment:
`# [build_platform != target_platform]` is for [cross-compiling](https://conda-forge.org/docs/maintainer/knowledge_base/#cross-compilation) which we are not doing in bioconda, so those can be omitted.

Reply:
So I guess `crossenv` and `cross-python_{{ target_platform }}` can also be removed?

Reply:
I believe so!

Reply:
Oh nice. Yeah sorry I just essentially copied that stuff blindly from a conda-forge recipe.